[2025-02-22T16:42:47.447+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-22T16:42:47.480+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: london_weather_pipeline.transform_weather_data manual__2025-02-22T16:42:44.181670+00:00 [queued]>
[2025-02-22T16:42:47.490+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: london_weather_pipeline.transform_weather_data manual__2025-02-22T16:42:44.181670+00:00 [queued]>
[2025-02-22T16:42:47.491+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-02-22T16:42:47.510+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): transform_weather_data> on 2025-02-22 16:42:44.181670+00:00
[2025-02-22T16:42:47.518+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=3024) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-02-22T16:42:47.519+0000] {standard_task_runner.py:72} INFO - Started process 3025 to run task
[2025-02-22T16:42:47.520+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'london_weather_pipeline', 'transform_weather_data', 'manual__2025-02-22T16:42:44.181670+00:00', '--job-id', '10', '--raw', '--subdir', 'DAGS_FOLDER/london_weather_pipeline/london_weather_etl.py', '--cfg-path', '/tmp/tmpd7jk745d']
[2025-02-22T16:42:47.521+0000] {standard_task_runner.py:105} INFO - Job 10: Subtask transform_weather_data
[2025-02-22T16:42:47.589+0000] {task_command.py:467} INFO - Running <TaskInstance: london_weather_pipeline.transform_weather_data manual__2025-02-22T16:42:44.181670+00:00 [running]> on host ee2081ae3a04
[2025-02-22T16:42:47.714+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='london_weather_pipeline' AIRFLOW_CTX_TASK_ID='transform_weather_data' AIRFLOW_CTX_EXECUTION_DATE='2025-02-22T16:42:44.181670+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-02-22T16:42:44.181670+00:00'
[2025-02-22T16:42:47.715+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-02-22T16:42:47.718+0000] {python.py:240} INFO - Done. Returned value was: {'latitude': '51.5074', 'longitude': '-0.1278', 'temperature': 13.0, 'windspeed': 13.9, 'winddirection': 253, 'weathercode': 2}
[2025-02-22T16:42:47.757+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-02-22T16:42:47.758+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=london_weather_pipeline, task_id=transform_weather_data, run_id=manual__2025-02-22T16:42:44.181670+00:00, execution_date=20250222T164244, start_date=20250222T164247, end_date=20250222T164247
[2025-02-22T16:42:47.814+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-02-22T16:42:47.871+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-02-22T16:42:47.876+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
